app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -3212,7 +3212,7 @@ class BrowserTabFragment :
                 },
             )
             lifecycleScope.launch {
-                webViewClient.configureWebView(
+                viewModel.configureWebView(
```

Comment:
Technically I don't need to move this, but this way I can inject the necessary plugins on the ViewModel only